src/main/java/Calculator.java

```diff
@@ -0,0 +1,21 @@
+public class Calculator {
+
+    public int add(int num1, int num2) {
+        return num1 + num2;
+    }
+
+    public int subtract(int num1, int num2) {
+        return num1 - num2;
+    }
+
+    public int multiply(int num1, int num2) {
+        return num1 * num2;
+    }
+
+    public int divide(int dividend, int divisor) {
+        if(divisor == 0){
+            throw new ArithmeticException("0으로 나눌 수 없습니다.");
+        }
```

Comment:
if문과 소괄호 사이에 공백을 두는 것이 코드 컨벤션입니다!
저도 처음엔 많이 놓쳤는데, 자동 포맷팅 단축키를 활용하면 편하더라고요! 참고하시면 좋을 것 같아요😊

```suggestion
        if (divisor == 0) {
            throw new ArithmeticException("0으로 나눌 수 없습니다.");
        }
```

Reply:
자동 포맷팅 단축키가 있다는 걸 몰랐는데, 덕분에 알게 되었습니다!
이제 자동 포맷팅 단축키를 활용하면 코드 컨벤션을 더욱 쉽게 맞출 수 있을 것 같아요. 감사합니다!